packages/app-project/Dockerfile

```diff
@@ -1,7 +1,20 @@
 FROM zooniverse/front-end-monorepo:latest
```

Comment:
This image is rebuilt on each production and staging build (see #1482), so can't be relied upon for local dev. Local dev should use a base image that's built from your local copy of the monorepo.

Reply:
Yep - longer term for the dev / testing workflow we should look at how we can use a different Dockerfile to use locally built images vs relying on this boostrap image. 

This is used in the deployment builds so it's going to rely on that bootstrap image from the jenkins CI run. We can certainly revisit how this works but short term i'd like to leave this as is.

Reply:
OK. I've got the same questions here as I had with the content pages app. If I'm making changes to the shared libs (eg. classifier, zooniverse react components or zoo theme) as part of a PR, will those changes be reflected in a local docker-compose build?

Reply:
So these changes are for deployment / CI pipeline not for local dev work, as the only system using these package dockerfiles are the CI tools.

Testing changes / builds locally will still take place via the root docker-compose directives and the root dockerfile: 
- https://github.com/zooniverse/front-end-monorepo/blob/2234b9cae3f0fc967d430e1016b8636564b8a083/docker-compose.yml#L15
- https://github.com/zooniverse/front-end-monorepo/blob/2234b9cae3f0fc967d430e1016b8636564b8a083/Dockerfile#L22

> If I'm making changes to the shared libs (eg. classifier, zooniverse react components or zoo theme) as part of a PR, will those changes be reflected in a local docker-compose build?

If you use each packages docker-compose files they will **not** include your changes as the base image it's building off is the CI image not the locally built images. 

Instead for dev work / testing you can use the root docker-compose / dockerfile to rebuild the base image and each app package (as linked above which includes the bootstrap step for building react components and lib classifier).